qemu/tests/virtio_serial_throttling.py

```diff
@@ -0,0 +1,60 @@
+import logging
+from virttest import error_context, env_process
+from virttest.utils_virtio_port import VirtioPortTest
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Induce throttling by opening chardev but not reading data from it:
+    1) Start guest with virtio-serial with unix option
+    2) On the host, open the chardev but don't read from it
+    3) In the guest, write to the virtio-serial port.
+    4) transfer data from host to guest but do not read in guest
+    5) repeat step 1 to 4 with tcp option.
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    @error_context.context_aware
+    def send_data_from_guest_to_host():
+        session = vm.wait_for_login()
+        port.open()
+        cmd = 'dd if=/dev/zero of=/dev/virtio-ports/%s bs=1024 count=1' % port.name
```

Comment:
@michiboo @nanliu-r  we miss windows guest before, if consider windows guest, try:
For windows guest,
  ```                                            
vport_name = '\\\\.\\Global\\' + port.name                       
from windows_support import WinBufferedReadFile                     
_vport = WinBufferedReadFile(vport_name)
_vport.write(message)
```

For linux guest, to keep consistent with windows guest, refer:
```
vport_name = '/dev/virtio-ports/%s' %port.name
_vport = os.open(vport_name, os.O_RDWR)
os.write(_vport, message) 
```

Reply:
from windows_support import WinBufferedReadFile   is not supported on linux host

Reply:
> from windows_support import WinBufferedReadFile is not supported on linux host

This is not a blocker, run it with windows only is ok.

Reply:
I changed to use dd instead in window